### DISCUSSION TITLE
Is there any way to commit with --no-verify without WIP?

### DISCUSSION THREAD

**Discussion by TrueWill** [jesseduffield/lazygit#2961] [Q&A]
Our workflow allows bypassing pre-commit hooks in certain cases.

`commitChangesWithoutHook` ('w' default shortcut) commits changes this way, but seems to only work if the description is "WIP". If I change that, it doesn't add `--no-verify`.

Is there any way to bypass hooks with any message?

I tried configuring
```
git:
  skipHookPrefix: ''
```
but got an error that that's required.

**Comment by mark2185** [comment]
Could you try invoking a custom command by running `:git commit --no-verify`?

I remember there was talk about revamping the prefix approach, but I don't think anything has taken root yet.

**Reply by TrueWill** [reply]
Sure, but I'd be typing that a lot. 😉 

For now I'm using `skipHookPrefix: '-'` which is adequate, but it would be nice to be able to omit the prefix.

**Reply by stefanhaller** [reply]
I think the idea was not to type this every time, but to configure a [custom command](https://github.com/jesseduffield/lazygit/blob/master/docs/Custom_Command_Keybindings.md) so that you can invoke it with a single keystroke.

**Comment by vikyathshirva** [comment]
Any update here ? wanted to know if we can add different types of args on commit

**Comment by allpepper** [comment]
any news on this one?

**Reply by stefanhaller** [reply]
Yes, see #4374. This has been available since version 0.49.
